README.md

```diff
@@ -1,35 +1,108 @@
-# Azure Functions PowerShell New Programming Model (Extension Bundles)
+# Azure Functions PowerShell Insider Preview of the v2 Programming Model
```

Comment:
Nit: Could we make it consistent throughout the doc whether we use v2 or V2?

Reply:
good catch, just made these changes. to align with Functions we should use v2 unless it's part of a title, then we can use V2.